technology/java-activemq-quarkus/run.sh

```
@@ -53,6 +53,16 @@ docker-compose up > target/activemq.log 2>&1 &
 readonly PID_DOCKER_AMQ=$!
 echo "Running the ActiveMQ broker via docker."
 
+# Wait a moment for ActiveMQ container to start.
+sleep 0.5
+
+# Check if the docker container with ActiveMQ broker is running.
+if ! kill -0 $PID_DOCKER_AMQ &> /dev/null ; then
```

In case the ActiveMQ docker container run fails (e.g. when docker is not available), we fail fast.